.circleci/config.yml

```diff
@@ -14,3 +14,8 @@ workflows:
           artifact-path: packed/orb.yml
           requires:
             - orb-tools/lint
+      - orb-tools/publish-dev:
+          orb-name: agileware-jp/redmine-plugin-orb
```

Comment:
It failed because the namespace `agileware-jp/redmine-plugin-orb` is not created yet.

> Error: the 'redmine-plugin-orb' orb does not exist in the 'agileware-jp' namespace. Did you misspell the namespace or the orb name?
https://circleci.com/gh/agileware-jp/redmine-plugin-orb/23#artifacts/containers/0

I don't have a authorization to run `circleci orb create agileware-jp/redmine-plugin-orb`.

> Note: Namespaces cannot be removed or renamed once you have claimed a namespace.
> Note: Only organization owners can create namespaces.
https://circleci.com/docs/2.0/creating-orbs/

@fkdkent Would you be able to create a namespace?

`circleci` command can be installed by `brew install circleci`.

Reply:
I've just created `agileware-jp/redmine-plugin`. (`-orb` suffix is not used for orb identifiers as far as I can see in https://circleci.com/orbs/registry/)

Reply:
Thanks! I updated the orb-name at 5f5692d